vt/emulate.go

```
@@ -201,36 +285,40 @@ func NewEmulator(be Backend) Emulator {
+	inb            func(byte)    // input byte function (faster than state switch)
+	style          Style
+	defaultStyle   Style
+	utfLen         int
+	pos            Coord
+	buffering      uint         // reference count - number of (re-entrant) buffering calls
+	autoWrap       bool         // next character will wrap (auto margin, deferred until char emitted)
+	sevenOnly      bool         // only allow 7-bit escapes (needed for KOI8, ShiftJIS, etc.)
+	appKeyPad      bool         // use application key pad keys?
+	name           string       // name of this emulator (used for extended attributes)
+	vers           string       // version string of this emulator (used for extended attributes)
+	savedPos       Coord        // saved via DECSC
```

golangci-lint (2.11.4)

[error] field savedPos is unused

(unused)